packages/build/src/error/build.js

```diff
@@ -2,11 +2,9 @@
 
 const safeJsonStringify = require('safe-json-stringify')
 
-const { addErrorInfo, getErrorInfo } = require('./info')
-
 // Retrieve error information from child process and re-build it in current
 // process. We need this since errors are not JSON-serializable.
-const jsonToError = function ({ name, message, stack, info = {}, errorProps = {} }) {
+const jsonToError = function ({ name, message, stack, ...errorProps }) {
```

Comment:
This file converts error instances to/from JSON. This is used to communicate error objects between the plugin child processes and the main parent process.